### ISSUE TITLE
Fix spacing of learn page on smaller screens

### ISSUE THREAD

**Issue by lindaokorie27** [ocaml/v2.ocaml.org#1374]
The spacing of the learn landing page sections on tablet and mobile is quite tight. This affects the legibility of different sections and blacks of text.

See images of desktop and mobile views

![spacing issue 2](https://user-images.githubusercontent.com/38317939/113411944-fc373080-93ae-11eb-9a8e-3f245d39a34e.JPG)
![spacing issue 1](https://user-images.githubusercontent.com/38317939/113411983-107b2d80-93af-11eb-9329-50ae6c5f5d96.JPG)

**Steps to recreate issue:**

View the page on browser device emulator with screen sizes 768px * 1024px and 375px * 812px

**Suggestions to solve**

On mobile view, I suggest padding be added above and below the sections to create the appearance of spacing. On tablet views, I suggest reorganizing the present content to take a two column layout. This might involve breaking the row with the "Industrial Users", "Success Stories" and "Online Course,Slides & Videos" into separate rows which will be a three-column layout on large screens, two-column on medium sized screens and single column on small screen layout.

On my check through the issues, I didn't see anything that handled this specific problem. 

If it's alright, I can work on it.

**Comment by pitag-ha** [comment]
Sounds good to me. I think you can start working on this, if you'd like :)

**Comment by gs0510** [comment]
@LindieK  are you working on this? :)

**Comment by lindaokorie27** [comment]
@gs0510 yes I am

**Comment by lindaokorie27** [comment]
@pitag-ha, @gs0510 I'm almost done with the changes here but I noticed the project is using bootstrap v2.3.1 are there plans to upgrade to a higher version?

**Comment by pitag-ha** [comment]
I don't think there are any plans for that. Is the bootstrap version an issue here?

**Comment by lindaokorie27** [comment]
Yes. Although I can make the changes to the page required using bootstrap this would entail writing more CSS classes increasing the size of the code base and the amount of work to be done maintaining the code.

There have been so many improvements made in bootstrap 3 and 4 that would make styling more efficient and neat. There are also many changes that would help in improving accessibility of the site and reduce the amount of work to be done and relevance of code written greatly.

If it can be looked into, it would greatly improve not just this issue but every other issue as regards to accessibility of the site.

**Comment by pitag-ha** [comment]
Yes, I think you're totally right! Sadly, upgrading bootstrap would be a huge amount of work, both due to the breaking changes in bootstrap and due to the changes we've made to our vendored version of it. If you want, you can give it a try anyways (file a different issue in that case), but I don't know how much time it would take - probably quite a bit.

For this issue, I think you can add the necessary CSS classes.

**Comment by lindaokorie27** [comment]
Okay I will finish this up and look into that.

**Comment by lindaokorie27** [comment]
@pitag-ha I have made the changes to the layout. Due to the structuring of the html, I was not able to achieve a two-column layout as I intended. Achieving it would mean restructuring the sections of the page and creating new styling for it. 

The present layout was being handled by one bootstrap class which is used on a lot of pages. To avoid breaking those pages, I created a class specific to the learn page and worked with that to make the columns responsive. So now it switches between a single column layout on  tablets and lower and a three-column layout on screens wider than that.

I also took time to go through the bootstrap file to see the changes made to the file. I also tested out bootstrap 4 to see how much would be affected by the change. Well all I can say is that it broke enough for me to appreciate the present bootstrap file as it is. ;)

**Comment by pitag-ha** [comment]
Sounds all good, @LindieK, thanks! :)
